packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -24,45 +24,64 @@
           data-testid="files-collaborators-no-reshare-permissions-message"
           v-text="noResharePermsMessage"
         />
-        <template v-if="$_ownerAsCollaborator">
-          <p id="original-sharing-user" v-translate class="oc-invisible-sr">File owner</p>
-          <show-collaborator
-            :collaborator="$_ownerAsCollaborator"
-            aria-describedby="original-sharing-user"
-          />
-          <hr />
-          <show-collaborator :collaborator="$_currentUserAsCollaborator" />
-        </template>
-        <template v-else>
-          <p id="collaborator-as-fileowner" v-translate class="oc-invisible-sr">
-            You are the file owner
-          </p>
-          <show-collaborator
-            :collaborator="$_currentUserAsCollaborator"
-            aria-describedby="collaborator-as-fileowner"
-          />
-        </template>
-        <hr v-if="collaborators.length > 0" class="oc-mt-s oc-mb-s" />
-        <transition-group
-          id="files-collaborators-list"
-          class="uk-list uk-list-divider uk-overflow-hidden oc-m-rm"
-          :enter-active-class="$_transitionGroupEnter"
-          :leave-active-class="$_transitionGroupLeave"
-          name="custom-classes-transition"
-          tag="ul"
-          :aria-label="$gettext('Share receivers')"
+        <div
+          class="avatars-wrapper"
+          :aria-label="sharedWithTooltip"
+          data-testid="shared-with-avatars"
+          @click="sharedWithClick"
         >
-          <li v-for="collaborator in collaborators" :key="collaborator.key">
+          <h2 class="shared-with-label" v-text="sharedWithLabel" />
+          <oc-avatars
```

Comment:
No clue how the CI got _that green_ if you're referencing a component the current version of ODS doesn't even know about 🤔

Reply:
oh that is true lol